source/developers/how-tos/enable-translations-new-repo.rst

```diff
@@ -183,35 +177,41 @@ Testing translations sync in your fork
 Before submitting a pull request for review in the `openedx-translations repo`_, you should test the workflow
 on a fork by following the steps below:
 
-#. Fork the `openedx-translations repo`_.
-#. Make a pull request to your fork and modify the `extract-translation-source-files.yml`_ workflow to use your
-   repo, your organization name and the branch in the clone section.
-   For example, the `frontend-lib-special-exams testing pull request`_ uses the ``Zeit-Labs/frontend-lib-special-exams`` repository with the branch
-   set via ``ref: fix-i18n``.
+#. Add the ``make extract_translation`` into your fork of the new repository e.g. ``Zeit-Labs/credentials`` in a new branch e.g. ``omar/add-pull-translations``
+#. Fork the `openedx-translations repo`_ e.g. ``Zeit-Labs/openedx-translations``.
+#. In your fork, modify the `extract-translation-source-files.yml`_ file in a new branch e.g. ``omar/add-credentials``.
+#. Go to the ``Actions`` tab in your repository (e.g. ``Zeit-Labs/openedx-translations``)
+#. From the left section, pick the `"Extract Translation Source Files" section in your fork`_
+#. Click on the "Run workflow" dropdown button with the following parameters:
 
-#. Modify the `extract-translation-source-files.yml`_ workflow to run ``pull_request`` events.
+   - **Use workflow from branch:** ``omar/add-credentials``
+   - **Repository to extract translation source files from:** ``credentials``
+   - **The ref to extract translation source files from:** ``omar/add-pull-translations``
+   - Click on the "Run workflow" button
 
-#. Verify that an automated source translation pull request is created on your fork similar to the
-   `chore - add updated translation source files`_ pull request.
+#. Verify the action ran successfully
+#. Verify the new automated branch e.g. ``automated/extract-translation-source-files-20230903T001829`` has been created with a new commit e.g. ``chore: add updated translation source files`` has been created
+
+Once all the above steps are verified, the extraction step is ready for use and the pull request has been tested.
+
+In order to test the ``make pull_translations`` step, please follow the steps below:
 
 #. Add any test translations to your fork of the `openedx-translations repo`_ in the repo directory to overcome the
    fact that translations don't exist in the upstream `openedx-translations repo`_ yet.
 
    We recommend copying existing translations. For example to test `credentials`_ we would copy the
    `course discovery translations`_ directory and modify it to include `credentials`_ conf/locale.
 
-#. Temporarily edit the ``Makefile`` so the ``pull_translations`` step pulls from your fork e.g.
-   ``atlas pull --repository=Zeit-Labs/openedx-translations``.
+#. Temporarily pull translations from the fork using the Makefile command e.g. ``make ATLAS_OPTIONS="--repository=Zeit-Labs/openedx-translations --revision=omar/add-credentials" pull_translations``
```

Comment:
Depending on the above comment should this be


```suggestion
#. Temporarily pull translations from the fork using the Makefile command e.g. ``make ATLAS_OPTIONS="--repository=Zeit-Labs/openedx-translations --revision=your-branch-name" pull_translations``
```

Reply:
`--repository=Zeit-Labs/openedx-translations` should probably also be updated to be `your-branch-name`